blueprints/blueprint-replacing-guacamole.md

```diff
@@ -65,6 +66,8 @@ In this case, I feel strongly that it will be easy enough to find alternatives t
 
 ## Loose Threads
 
-https://community.networkreliability.engineering/t/web-terminal-emulator-replacement/184
-https://community.networkreliability.engineering/t/guacamole-proxy-only-for-ssh-or-vnc-too/44
-- [Support VNC Connections Too](https://github.com/nre-learning/antidote-web/issues/65)
\ No newline at end of file
+These are some open discussions on this topic that will need to be linked here.
+
+- https://community.networkreliability.engineering/t/web-terminal-emulator-replacement/184
+- https://community.networkreliability.engineering/t/guacamole-proxy-only-for-ssh-or-vnc-too/44
+- [Support VNC Connections Too](https://github.com/nre-learning/antidote-web/issues/65)
```

Comment:
While it would be great to have VNC, stable SSH connections are a priority.  VNC support should not prevent us from moving forward with replacing Guacamole.